src/assets/toolkit/styles/components/sky.css

```diff
@@ -5,7 +5,7 @@
   --Sky-background-color-dark: color(var(--color-navy) h(+5) s(-12) l(+18));
   --Sky-color: var(--color-white);
 
-  --Sky-clouds-image: url("/images/sky-clouds.svg");
+  --Sky-clouds-image: svg-load("../../../../static/images/sky-clouds.svg");
```

Comment:
You might consider specifying the `path` option to simplify this.

See [here](https://github.com/TrySound/postcss-inline-svg#options) and [here](https://github.com/postcss/postcss-use#standard-syntax) for info.